GitCommands/PathUtil.cs

```diff
@@ -512,6 +512,12 @@ public static string FindInFolders(this string fileName, IEnumerable<string?> fo
                     return fullName;
                 }
 
+                fullName = FindFileInEnvVarFolder("LOCALAPPDATA", Path.Combine("Programs", location), fileName);
```

Comment:
I suggest this is moved to 503.
A user may not have control of Program Files but only local user.
Not sure if there are recommendations here.

Reply:
Done (with a small refactoring)